pkg/conversion/json.go

```diff
@@ -0,0 +1,6 @@
+package conversion
+
+import jsoniter "github.com/json-iterator/go"
+
+// TFParser is a json parser to marshal/unmarshal using "tf" tag.
```

Comment:
I'd suggest having two parsers here for `tf` and `json` tags and use those everywhere instead of `encoding/json`. `json-iterator/go` claims to be much faster than standard json library and it has some useful [capabilities](http://jsoniter.com/migrate-from-go-std.html) like retrieving a specific key from the JSON blob without casting it to anything.

Reply:
Done.